dhcpv6/duid.go

```diff
@@ -25,6 +25,14 @@ var DuidTypeToString = map[DuidType]string{
 	DUID_UUID: "DUID-UUID",
 }
 
+func (d DuidType) String() string {
+	dtype := DuidTypeToString[d]
```

Comment:
this function could be even simpler:
```
if dtype, ok := DuidTypeToString[d]; ok {
    return dtype
}
return "Unknown"
```

However I'm not sure if the default return value here should be "Unknown"

Reply:
That is what we are currently returning in https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/duid.go#L84

Reply:
as discussed in person, let's leave it as it is :)